lib/venue.rb

```diff
@@ -1,2 +1,29 @@
 class Venue
+  attr_reader :name,
+              :capacity,
+              :patrons
+
+  def initialize(name, capacity)
+    @name     = name
+    @capacity = capacity
+    @patrons  = []
+  end
+
+  def add_patron(name)
+    @patrons << name
+  end
+
+  def yell_at_patrons
+    @patrons.map do |patron|
+      patron.upcase
+    end
+  end
+
+  def over_capacity?
+    @patrons.length > @capacity
+  end
+
+  def kick_out
+    @patrons.pop
```

Comment:
This works but how can we check when we would need it to remove a patron or not? Could you use a `loop` here? So something like 

```
until over_capacity? == false
   @patrons.pop
end

while over_capacity? == true
    @patrons.pop
end
```